ytypes/schema_tests/diff_benchmark_test.go

```diff
@@ -0,0 +1,44 @@
+package validate
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/idefixcert/ygot/ygot"
```

Comment:
Can you update the import path here? Travis is failing based on this import being not found during the run.